.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
@franknoirot we'll super need to talk about this. If you disable this flag you will see some bad dependencies stuff at the npm install stage including xstate with the old inspect package. We likely need to upgrade a bunch of things to be 100% clean but that seemed like a good way to move forward